src/modules-load/modules-load.c

```diff
@@ -163,34 +179,44 @@ static int run(int argc, char *argv[]) {
         if (r < 0)
                 log_warning_errno(r, "Failed to parse kernel command line, ignoring: %m");
 
-        r = module_setup_context(&ctx);
-        if (r < 0)
-                return log_error_errno(r, "Failed to initialize libkmod context: %m");
+        int sockets[2];
+        const int max_tasks = MAX(MIN(sysconf(_SC_NPROCESSORS_ONLN) / 2, 10), 2);
```

Comment:
`sysconf()` can actually fail, in realistic cases (it goes to /sys/), hence please catch that error.

alos, we don't use "const" on local variables (unless combined with static, i.e. truly immutable).

Also, sysconf() returns "long". Here you store its result int an "int". On 32bit archs you are thus silenlty dropping 32bit here. please don't, i.e. stick to "long"